src/routes.js

```diff
@@ -735,15 +735,6 @@ const routes = [
       restoreSession: true,
       requireLogin: false,
       title: () => [form.nameOrId]
-    },
-    // onceEnketoId can be used by only public-links as it doesn't support offline mode
-    beforeEnter: (to) => {
-      const { enketoId, actionType } = to.params;
-
-      if (enketoId.length === 64 && actionType === 'offline') {
```

Comment:
Could you say more about why we don't need this check anymore?

Would it be worth moving the check to `FormSubmission`? Something like:

```js
watchEffect(() => {
  if (form.dataExists && props.enketoId === form.enketoOnceId && props.actionType === 'offline')
    router.push('/');
});
```

Reply:
I removed it as it was not adding any value, if somebody tries to access offline mode with enketoOnceId, then Enketo will throw 404 error